proposals/00x-skips/README.md

```diff
@@ -0,0 +1,170 @@
+README files and the website are the primary sources of documentation for SpinKube.
+
+Moving forward, I propose that we establish a set of principles for how we document SpinKube and its
+projects using the current system as a framework. These principles should be followed by all
+existing and future projects in the SpinKube ecosystem.
+
+### Prinicples
+
+The following describes high-level principles that should be followed when documenting SpinKube:
+
+1. We place high importance on the consistency and readability of our documentation.
+1. We use a consistent writing style across all projects.
+1. We structure documentation in a consistent manner.
+1. We keep links and references up-to-date.
+1. We ensure that documentation matches the version of the project it describes.
+
+### Writing style
+
+Documentation should be written in American English. This includes spelling, grammar, and
+punctuation. For example, use "color" instead of "colour", "center" instead of "centre", and "gray"
+instead of "grey".
+
+Use the active voice instead of the passive voice. For example, use "SpinKube creates a new SpinApp"
+instead of "A new SpinApp is created by SpinKube".
+
+Use the present tense instead of the past tense. For example, use "SpinKube creates a new SpinApp"
+instead of "SpinKube created a new SpinApp".
+
+Use the imperative mood instead of the indicative mood. For example, use "Run `spin kube scaffold`"
+instead of "You can run `spin kube scaffold`".
+
+Use the second person instead of the first person. For example, use "You can run `spin kube
+scaffold`" instead of "I can run `spin kube scaffold`".
+
+When using pronouns in reference to a hypothetical person, such as "a user with a session cookie",
+gender-neutral pronouns (they/their/them) should be used. Instead of:
+
+- he or she... use they.
+- him or her... use them.
+- his or her... use their.
+- his or hers... use theirs.
+- himself or herself... use themselves.
+
+Try to avoid using words that minimize the difficulty involved in a task or operation, such as
+"easily", "simply", "just", "merely", "straightforward", and so on. People's experience may not
+match your expectations, and they may become frustrated when they do not find a step as
+"straightforward" or "simple" as it is implied to be.
+
+Try to avoid using acronyms and jargon that are not widely known. For example, use "Kubernetes"
+instead of "k8s", or "Custom Resource Definition" instead of "CRD". If you must use an acronym or
+jargon, define it the first time it is used in the document, and include a link to the glossary
+section of the document.
+
+Sentences should be hard-wrapped at 100 characters. This makes it easier to read and edit the
+documentation in a text editor.
+
+In section titles, capitalize only initial words and proper nouns. For example, use "How to install
+SpinKube" instead of "How to Install SpinKube".
+
+Here are some style guidelines on commonly used terms throughout the documentation:
```

Comment:
I will add a few more: 

- containerd instead containerD
- Wasm instead of WASM